### PR TITLE
refactor(protocol): changes related to the HTTP APIs

### PR DESCRIPTION
**Pull request by maximerety**
Contains also misc typo fixes.

### CODE REVIEW

File: protocol.md [hunk 1]
```diff
@@ -59,39 +59,41 @@ If no [Local Encrypted Storage] is found, *Tanker Core* will create one automati
...
+
+A second `device_creation` block is created with the *physical device*'s [Device Encryption Key Pair] and [Device Signature Key Pair]. *Tanker Core* creates a new ephemeral key pair, signs this block with the ephemeral private key and uses the *virtual device*'s [Device Signature Key Pair] to sign the *delegation* and fill the block's *delegation_signature* field.
 
 Finally, both `device_creation` *block*s are pushed to the *Trustchain*, the *virtual device* before the *physical one*, and the [Verification Key] is uploaded to the *Tanker server* according to the [Verification Method] used by the *user*.
 
-Assuming the pushed *block*s are correct, the *Tanker server* acknowledges them, allowing the just created *device* to proceed and [authenticate itself](#device-authentication).
+Assuming the pushed *block*s are correct, the *Tanker server* stores them, and initiates an authenticated session for the *physical device*. The just created *physical device* permanently stores its [Device Encryption Key Pair] and [Device Signature Key Pair] in the [Local Encrypted Storage].
```
Comment (blastrock) [inline protocol.md:74]: Aren't they stored before the blocks are sent to the server?

File: protocol.md [hunk 2]
```diff
@@ -106,27 +108,29 @@ The *device* initiates the authentication by requesting an *authentication chall
...
+- The signature of the challenge with the private [Device Signature Key Pair]
 
 This allows the *Tanker server* to check that:
 
 - The *device* is registered on the *Trustchain*
 - The *device* is correctly associated with the provided [User ID]
 - The signature matches the challenge
 
-If any of these check fail, the connection is closed.
+If any of these check fail, the authenticated session is not created.
```
Comment (blastrock) [inline protocol.md]: *checks

**Review by blastrock — CHANGES_REQUESTED** [review]
We should update the block-verification too, no?

File: protocol.md (hunk 1 above)
Comment (maximerety) [reply protocol.md:74]: Not anymore! (in a very close future TM)

Comment (blastrock) [reply protocol.md:74]: Hmm I see ^^

**Review by blastrock — APPROVED** [review]

**Pull request merged**